peerloop/domain/user/views.py

```diff
@@ -49,3 +52,15 @@ async def verify_email(
     request: VerifyEmailRequest, user_service: UserService = Depends(Provide["user_container.user_service"])
 ) -> None:
     await user_service.verify_email(email=request.email, verification_code=request.verification_code)
+
+
+@router.get("/me", status_code=status.HTTP_200_OK, response_model=UserResponse)
+@inject
+async def get_me(user: Annotated[User, Depends(get_current_user)]) -> User:
+    return user
+
+
+@router.get("/verified-me", status_code=status.HTTP_200_OK, response_model=UserResponse)
+@inject
+async def get_verified_me(user: Annotated[User, Depends(get_current_verified_user)]) -> User:
+    return user
```

Comment:
Why do we need these two routes? I wonder if they're for testing purposes.

Reply:
Yes, I made them in order to test `get_current_user` and `get_current_verified_user` in swagger, so these endpoints are unnecessary except for test purposes!

`/me` endpoint is about "log in" and `/verified-me` endpoint is about "log in" & "email verification". 

Would it be appropriate to remove these routes?